### PR TITLE
Opaque glossary

### LINKED ISSUE #348: Clarify "opaque"
The term *opaque* is used several times in the specification:

https://www.w3.org/TR/2019/CR-trace-context-20190813/#relationship-between-the-headers
https://www.w3.org/TR/2019/CR-trace-context-20190813/#tracestate-header-field-values
https://www.w3.org/TR/2019/CR-trace-context-20190813/#value
https://www.w3.org/TR/2019/CR-trace-context-20190813/#privacy-of-tracestate-field

Nevertheless, the meaning of this term in these situations may not be very easy to understand to readers that are less familiar with the document. A explanation would be very useful.

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Fixes #348

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/pull/434.html" title="Last updated on Nov 3, 2020, 8:17 PM UTC (cedf261)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/434/6d95bd9...cedf261.html" title="Last updated on Nov 3, 2020, 8:17 PM UTC (cedf261)">Diff</a>

### CODE REVIEW

**Review by mtwo — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -114,7 +114,15 @@ <h1>Glossary</h1>
       that handled the chain of requests initiated by this button
       being pressed.
     </dd>
-  </dl>
+    </dl>
+    <dl>
+    <dt><dfn data-lt="opaque">Opaque value</dfn></dt>
+    <dd>
+      An opaque value refers to the value that can only be understood,
```
Comment (arminru) [inline index.html]:
```suggestion
      An opaque value refers to a value that can only be understood,
```

File: index.html [hunk 2]
```diff
@@ -114,7 +114,15 @@ <h1>Glossary</h1>
...
+      or processed in any way by the distributed trace participant generated
```
Comment (arminru) [inline index.html]:
```suggestion
      or processed in any way by the distributed trace participant that generated
```

**Review by arminru — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -114,7 +114,15 @@ <h1>Glossary</h1>
       that handled the chain of requests initiated by this button
       being pressed.
     </dd>
-  </dl>
+    </dl>
+    <dl>
+    <dt><dfn data-lt="opaque">Opaque value</dfn></dt>
+    <dd>
+      An opaque value refers to a value that can only be understood,
```
Comment (dyladan) [inline index.html]: Incorrect comma use.

Per [purdue online writing lab notes on comma use](): 

>13. Don't put a comma between the two verbs or verb phrases in a compound predicate.
>
> INCORRECT: INCORRECT: We laid out our music and snacks, and began to study.
> CORRECT: INCORRECT: We laid out our music and snacks and began to study.

```suggestion
      An opaque value refers to a value that can only be understood
```

File: index.html [hunk 4]
```diff
@@ -114,7 +114,15 @@ <h1>Glossary</h1>
...
+      or processed in any way by the distributed trace participant that generated
+      this value. Any other participant must treat it as a blob of bytes.
```
Comment (dyladan) [inline index.html]: I think the definition is a little too editorial/wordy, and contains the word "must" which may be misconstrued as normative. I think a better definition which assumes less would be this:

```suggestion
      An opaque value refers to a value with unspecified meaning
      or format beyond the limits defined in the specification.
```

**Review by danielkhan — APPROVED** [review]

File: index.html (hunk 4 above)
Comment (SergeyKanzhelev) [reply index.html]: I agree with your evaluation. Unfortunately PR already merged with this definition. Would you like to submit your wording as a proposal? Or let me know and I'll do it

**Pull request merged**